koji/README.md

```diff
@@ -0,0 +1,80 @@
+---
+title: koji
+homepage: https://github.com/its-danny/koji
+tagline: |
+  🦊 An interactive CLI for creating conventional commits.
+---
+
+To update or switch versions, run `webi koji@stable` (or `@v2`, `@beta`, etc).
+
+**Note**: You should install git before installing koji.
+
+## Files
+
+These are the files that are created and/or modified with this installer:
+
+```
+~/.config/envman/PATH.env
+~/.local/bin/koji
+~/.local/opt/koji-VERSION/bin/koji
+```
+
+## Cheat Sheet
+
+`koji` is an interactive CLI for creating
+[conventional commits](https://www.conventionalcommits.org/en/v1.0.0/), ran in
+place of `git commit`.
+
+### Using koji
+
+```bash
+cd dev/work-stuff
+# Do some work
+cd dev/work-stuff
+git add .env.production
```

Comment:
umm... no... DO NOT commit private keys for prod!!!

Reply:
joke from repo README lol, probably shoulda changed it